src/app/page.tsx

```diff
@@ -42,12 +42,9 @@ const HomePage = () => {
             </p>
             <p className="mb-6 text-lg">
               Voor meer uitleg over deze catalogus, zie{" "}
-              <a
-                href="https://catalogus-test.healthdata.nl/about"
-                className="text-blue-500 hover:underline"
-              >
+              <Link href="/about" className="text-blue-500 hover:underline">
                 Over de catalogus
-              </a>
+              </Link>
```

Comment:
I don't  understand the difference between Link and "a" ? Is it now not differently displayed

Reply:
https://reactrouter.com/en/main/components/link; The a  tag is hardcoded, link uses the app router. It's the same as used in the menu bar.